values/vault.yaml.gotmpl

```diff
@@ -0,0 +1,265 @@
+templates:
+- |
+  apiVersion: vault.banzaicloud.com/v1alpha1
+  kind: "Vault"
+  metadata:
+    name: "vault"
+  spec:
+    size: 1
+    image: vault:1.6.0
+    bankVaultsImage: banzaicloud/bank-vaults:1.8.0
+
+    # Common annotations for all created resources
+    annotations:
+      common/annotation: "true"
+
+    # Vault Pods , Services and TLS Secret annotations
+    vaultAnnotations:
+      type/instance: "vault"
+
+    # Vault Configurer Pods and Services annotations
+    vaultConfigurerAnnotations:
+      type/instance: "vaultconfigurer"
+
+    # Vault Pods , Services and TLS Secret labels
+    vaultLabels:
+      example.com/log-format: "json"
+
+    # Vault Configurer Pods and Services labels
+    vaultConfigurerLabels:
+      example.com/log-format: "string"
+
+    # Specify the ServiceAccount where the Vault Pod and the Bank-Vaults configurer/unsealer is running
+    serviceAccount: vault
+
+    # Specify the Service's type where the Vault Service is exposed
+    # Please note that some Ingress controllers like https://github.com/kubernetes/ingress-gce
+    # forces you to expose your Service on a NodePort
+    serviceType: ClusterIP
+
+    vaultInitContainers:
+      - name: orchestrate-plugin
+        image: alpine:3.12
+        command:
+          - /bin/sh
+          - -c
+          - |
+            apk add --no-cache jq wget
+            echo "Installing orchestrate-hashicorp-vault-plugin version '$PLUGIN_VERSION'"
+            parser=". | map(select(.tag_name == \"$PLUGIN_VERSION\"))[0].assets | map(select(.name == \"$PLUGIN_FILE\"))[0].id"
+
+            echo "Obtaining asset_id of $PLUGIN_FILE..."
+            asset_id=`wget -q --auth-no-challenge --proxy off --header="Authorization: token $GITHUB_TOKEN" --header="Accept: application/vnd.github.v3.raw" https://api.github.com/repos/$PLUGIN_REPO/releases -O - | jq "$parser"`
+            if [ -z "$asset_id" ]; then
+              echo "ERROR: version $PLUGIN_VERSION not found asset $PLUGIN_FILE"
+              exit 1
+            fi;
+
+            echo "Downloading $FILE(ID:$asset_id)..."
+            wget -q --auth-no-challenge --proxy off --header='Accept:application/octet-stream' \
+              https://$GITHUB_TOKEN:@api.github.com/repos/$PLUGIN_REPO/releases/assets/$asset_id \
+              -O "${PLUGINS_PATH}/orchestrate"
+
+            echo "File at ${PLUGINS_PATH}/orchestrate"
+            chmod +x ${PLUGINS_PATH}/orchestrate
+        env: 
+          - name: GITHUB_TOKEN
+            value: {{ .Environment.Values.vaultPlugin.token }}
+          - name: PLUGIN_VERSION
+            value: {{ .Environment.Values.vaultPlugin.tag }}
+          - name: PLUGIN_FILE
+            value: {{ .Environment.Values.vaultPlugin.file }}
```

Comment:
Why you want to make this value customizable, we can decide over users where is best to store the pluging and load it.